src/main/java/com/oltpbenchmark/benchmarks/templated/util/TemplatedValue.java

```diff
@@ -0,0 +1,112 @@
+package com.oltpbenchmark.benchmarks.templated.util;
+
+import java.util.Random;
+
+import com.oltpbenchmark.distributions.ScrambledZipfianGenerator;
+import com.oltpbenchmark.distributions.ZipfianGenerator;
+
+/**
+ * This class is used to store information about the values
+ * used in templated benchmarks. It can hold static values but also
+ * generators for value distributions
+ */
+public class TemplatedValue {
+    ValueGenerator distribution;
+    Long min;
+    Long max;
+    Long seed;
+    String value;
+    String minS;
+    String maxS;
```

Comment:
Why store `minS` and `maxS` from just the `min` and `max` `long` values?

Can't you just `toString()` those as necessary?

Or was this intended to support `min` `max` string for String type distributions?
In that case, you'd need to always pass the String value in instead correct?
Maybe this is a case for a generic class that abstracts the internal type stored (e.g., `TemplatedValue<Long>` vs. `TemplatedValue<String>`